ui/raidboss/data/03-hw/dungeon/sohm_al.txt

```diff
@@ -0,0 +1,141 @@
+319.9 "Acid Rain" sync /:Raskovnik:ED7:/
+328.5 "Phytobeam" sync /:Raskovnik:ED5:/
+333.7 "Bloody Caress" sync /:Raskovnik:ED1:/
+# Loop
+342.8 "Leafstorm" sync /:Raskovnik:ED6:/ jump 303.0
+348.0 "Bloody Caress" # sync /:Raskovnik:ED1:/
+354.1 "--sync--" # sync /:Raskovnik:ED2:/
+359.7 "Acid Rain" # sync /:Raskovnik:ED7:/
+368.2 "Phytobeam" # sync /:Raskovnik:ED5:/
+373.4 "Bloody Caress" # sync /:Raskovnik:ED1:/
+
+# jump back to group 1
+500.0 "--sync--" sync /:1395:Raskovnik/ window 500,0 jump 103.0
+
+
+### Myath
+# -ic "Chyme Of The Mountain"
+
+1000.0 "Start" sync /00:0839:The Wound will be sealed off/ window 1000,1
+
+1006.1 "Third Leg Forward" sync /:Myath:1382:/
+1011.2 "Overbite" sync /:Myath:EDB:/
+1019.3 "Razor Scales" sync /:Myath:EDC:/
+1024.5 "Third Leg Forward" sync /:Myath:1382:/
+1031.6 "Third Leg Forward" sync /:Myath:1382:/
+1036.7 "Overbite" sync /:Myath:EDB:/
+# Loop
+1044.8 "Razor Scales" sync /:Myath:EDC:/ jump 1019.3
+1050.0 "Third Leg Forward" # sync /:Myath:1382:/
+1057.1 "Third Leg Forward" # sync /:Myath:1382:/
+1062.2 "Overbite" # sync /:Myath:EDB:/
+1070.3 "Razor Scales" # sync /:Myath:EDC:/
+1075.5 "Third Leg Forward" # sync /:Myath:1382:/
+
+# Myath would use Primordial Roar twice:
+# 1. HP < 90%?
+# 2. HP < 50%? also summon Chyme Of The Mountain
+1200.0 "Primordial Roar" sync /:Myath:EE2:/ window 200,0
+1207.1 "Third Leg Forward" sync /:Myath:1382:/
+1214.2 "Ensnare" sync /:Myath:EDD:/
+1219.2 "Mad Dash" sync /:Myath:EE[01]:/
+1221.5 "Ensnare" sync /:Myath:EDD:/
+1226.5 "Mad Dash" sync /:Myath:EE[01]:/
+1229.0 "Ensnare" sync /:Myath:EDD:/
+1236.1 "Mad Dash" sync /:Myath:EE[01]:/
+1238.4 "Ensnare" sync /:Myath:EDD:/
+1245.4 "Mad Dash" sync /:Myath:EE[01]:/
+
+# Jump back to loop
+1249.8 "Third Leg Forward" sync /:Myath:1382:/ window 30,5 jump 1006.1
+1254.9 "Overbite" # sync /:Myath:EDB:/
+1263.0 "Razor Scales" # sync /:Myath:EDC:/
+1268.2 "Third Leg Forward" # sync /:Myath:1382:/
+1275.3 "Third Leg Forward" # sync /:Myath:1382:/
+1280.4 "Overbite" # sync /:Myath:EDB:/
+
+
+### Tioman
+# -ic "Comet"
+
+2000.0 "Start" sync /00:0839:Hess Afah will be sealed off/ window 2000,1
+
+2006.0 "Abyssic Buster" sync /:Tioman:EE3:/
+2013.2 "Chaos Blast" sync /:Tioman:EE5:/
+2016.5 "--sync--" sync /:Tioman:EEB:/
+2020.4 "Abyssic Buster" sync /:Tioman:EE3:/
+2030.0 "Comet" sync /:Tioman:EE6:/
+2043.7 "Meteor Impact" sync /:Tioman:1387:/
+# Loop
+2047.3 "Abyssic Buster" sync /:Tioman:EE3:/ jump 2006.0
+2054.5 "Chaos Blast" # sync /:Tioman:EE5:/
+2057.8 "--sync--" # sync /:Tioman:EEB:/
+2061.7 "Abyssic Buster" # sync /:Tioman:EE3:/
+2071.3 "Comet" # sync /:Tioman:EE6:/
+2084.9 "Meteor Impact" # sync /:Tioman:1387:/
+
+# 00:0044:Tioman:Arrogant insects! Be crushed beneath the weight of my fury!
+2200.0 "Heavensfall" sync /:Tioman:EE7:/ window 200,0
+2223.5 "Dark Star" # sync /:Tioman:EE4:/
+2246.9 "Dark Star" # sync /:Tioman:EE4:/
+2270.5 "Dark Star" # sync /:Tioman:EE4:/
+# TODO: didn't see an Enrage?
```

Comment:
Do the dark stars keep happening?

Reply:
In my turn, yes.
We killed wings after the third Dark Stars because it really hurts, our Scholar was hardly to keep healing, so didn't see whether there is an Enrage or just keep using Dark Star.😂